pyonfx/convert.py

```diff
@@ -44,12 +58,12 @@ class ColorModel(Enum):
 
 class Convert:
     """
-    This class is a collection of static methods that will help
+    This class is a collection of methods that will help
     the user to convert everything needed to the ASS format.
     """
 
     @staticmethod
-    def time(ass_ms: Union[int, str]) -> Union[str, int, ValueError]:
+    def time(ass_ms: Union[int, str]) -> Union[str, int]:
```

Comment:
`time` method should now make use of `timems_to_assts` and `assts_to_timems`, right?

Reply:
Indeed, I pasted the old code and removed the use of these methods